src/parser/jobs/mnk/modules/GreasedLightning.js

```diff
@@ -299,7 +294,20 @@ export default class GreasedLightning extends Module {
 			}))
 		}
 
-		if (this._wastedWind > 0) {
+		if (missedEarth) {
+			this.suggestions.add(new Suggestion({
+				icon: ACTIONS.RIDDLE_OF_EARTH.icon,
+				content: <Fragment>
+					Check the fight timeline to see when you can save <StatusLink {...STATUSES.GREASED_LIGHTNING_I} /> with <ActionLink {...ACTIONS.RIDDLE_OF_EARTH} />.
```

Comment:
This isn't really... a _suggestion_. It's not saying what they could do better, nor what they shouldn't be doing - it's just saying "LOOK AT SOMETHING ELSE".

I realise there's an existing one above, but I'd frankly much rather all instances of it be updated.